### ISSUE TITLE
tenary loss

### ISSUE THREAD

**Issue by lovean1** [lhaippp/GyroFlow-PyTorch#8]
作者你好，我发现论文中用到了census loss这个损失函数，但是我在params.json文件中发现该损失函数的系数是0，请问这里是不是需要修改系数再微调

**Comment by lhaippp** [comment]
hi, Gyroflow needs at least 2 stages to get converge:
1. pretrain stage with simple photometric loss, i.e., weight of census loss = 0
2. extra specifically designed loss, e.g., cencus loss, occluson checking, etc. So we enable it in the second stage

**Comment by lovean1** [comment]
> hi, Gyroflow needs at least 2 stages to get converge:
> 
> 1. pretrain stage with simple photometric loss, i.e., weight of census loss = 0
> 2. extra specifically designed loss, e.g., cencus loss, occluson checking, etc. So we enable it in the second stage

感谢回答，请问能不能给出第二阶段的loss系数？是params中的0,0,1吗？

**Comment by lhaippp** [comment]
> 感谢回答，请问能不能给出第二阶段的loss系数？是params中的0,0,1吗？

yes, try it ;)
